client/checkout/woopay/email-input-iframe.js

```diff
@@ -76,7 +74,7 @@ export const handlePlatformCheckoutEmailInput = async (
 		( 'undefined' !== typeof performance &&
 			'back_forward' ===
 				performance.getEntriesByType( 'navigation' )[ 0 ].type ) ||
-		'true' === searchParams.get( 'skip_platform_checkout' );
+		'true' === searchParams.get( 'skip_woopay' );
 
 	// Track the current state of the header. This default
 	// value should match the default state on the platform.
```

Comment:
Shouldn't we change this as well?
```suggestion
	// value should match the default state on WooPay.
```